docs-source/content/userguide/managing-domains/domain-events.md

```diff
@@ -30,24 +30,26 @@ The operator generates these event types, which indicate the following:
  *  `DomainProcessingCompleted`:  The operator successfully completed the processing of a domain resource.
  *  `DomainProcessingAborted`:  The operator stopped processing a domain when the operator encountered a fatal error or a failure that persisted after the specified maximum number of retries.
  *  `DomainValidationError`:  A validation error or warning is found in a domain resource. Please refer to the event message for details.
+ *  `NamespaceWatchingStarted`: The operator has started to watch for domains in a namespace.
+ *  `NamespaceWatchingStopped`: The operator has stopped to watch for domains in a namespace.
```

Comment:
has started to watch -> has started watching
has stopped to watch -> has stopped watching